### PR TITLE
Reservations

### PR DESCRIPTION
**Pull request by looh16**
**Add reservations endpoints**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
   # root "articles#index"
   get '/current_user', to: 'users#logged_in_user'
 
-  resources :products, only: [ :create, :index, :destroy, :show ]
+  resources :products, only: [ :create, :index, :destroy, :show ] do
+    resources :reservations, only: [ :create ]
```
Comment (ishpaul777) [inline config/routes.rb:12]: Users should be able to see all reservation, please include `:index` to reservations resources.

**Review by ishpaul777 — CHANGES_REQUESTED** [review]
Kindlly make the requested change and you are good to merge the branch👍⚡

**Comment by looh16** [comment]
I don't know what happened, but I was pretty sure I added index. I already updated.

**Review by ishpaul777 — APPROVED** [review]
Every thing seems ok! Fell Free to merge!

**Pull request merged**